tests/ut/scargo_check/ut_copyright_checker.py

```diff
@@ -33,13 +35,12 @@
     indirect=True,
 )
 def test_check_copyright_pass(
-    mock_file_contents: MagicMock,
-    caplog: pytest.LogCaptureFixture,
-    config: Config,
-    mock_find_files: MagicMock,
+        mock_file_contents: MagicMock,
```

Comment:
Is it something your IDE does? Similar thing happened before. Also, since you were able to commit this, it means that you haven't installed pre-commit. Please do.